src/pages/appSearch.jsx

```diff
@@ -45,14 +44,7 @@ const AppSearchDae = () => {
     where('c_etat_fonct', '==', 'En fonctionnement'),
   )
 
-  //   useEffect(() => {
-  //     /** Tous les DAE **/
-
-  //     execute(getDocsCustom(ENDPOINT))
-  //   }, [execute])
-
   useEffect(() => {
-    /** Tous les DAE **/
     execute(initialQuery)
   }, [execute])
```

Comment:
@BambaThiam, 
Est-ce que tu as le warning suivant :
"React Hook useEffect has a missing dependency: 'initialQuery'. Either include it or remove the dependency array"
Si oui, supprime 'initialQuery' des dépendances via eslint ( // eslint-disable-next-line react-hooks/exhaustive-deps)